### DISCUSSION TITLE
disable version checks for github action

### DISCUSSION THREAD

**Discussion by jukzi** [eclipse-platform/.github#168] [Ideas]
I think its enough if the CI complains about missing version bumps or API changes. I don't need it tested on different platforms. May be it would even decrease github time
![image](https://github.com/eclipse-platform/.github/assets/51790620/975217cb-3b6a-433e-a420-89696aa6a807)

**Comment by laeubi** [comment]
I don't think it has any value to have diverging build scripts in this regard, if something is wrong one should not expect other thing to succeed. e.g. the CodeQL check for example also fails on compile errors...

**Comment by jonahgraham** [comment]
For CDT we run the version checks (and formatting checks) as a separate workflow so that the build & test of the content can complete so that contributors can know the only problem is the version (or formatting) error. e.g. we do `-P baseline-compare-and-replace` and `-P api-baseline-check` only on one job.

**Reply by jukzi** [reply]
@jonahgraham can you contribute such change, please?
